.gitignore

```diff
@@ -36,3 +36,6 @@ dist-ssr
 # next
 **/.next
 packages/frontend/out
+
+# test Level DB
+packages/stateManager/tmpDb
```

Comment:
use [memdb](https://github.com/Level/memdown) so that it will not create this folder in the first place